modules/example/src/smithy4s/example/TestItOut.scala

```diff
@@ -0,0 +1,19 @@
+package smithy4s.example
+
+import smithy4s.schema.Schema._
+
+case class TestItOut(age: Option[smithy4s.example.refined.Age] = None, personAge: Option[smithy4s.example.refined.Age] = None, fancyList: Option[smithy4s.example.refined.FancyList] = None, name: Option[smithy4s.example.refined.Name] = None)
+object TestItOut extends smithy4s.ShapeTag.Companion[TestItOut] {
+  val id: smithy4s.ShapeId = smithy4s.ShapeId("smithy4s.example", "TestItOut")
+
+  val hints : smithy4s.Hints = smithy4s.Hints.empty
+
+  implicit val schema: smithy4s.Schema[TestItOut] = struct(
+    Age.underlyingSchema.optional[TestItOut]("age", _.age),
+    PersonAge.underlyingSchema.optional[TestItOut]("personAge", _.personAge),
+    FancyList.underlyingSchema.optional[TestItOut]("fancyList", _.fancyList),
+    Name.underlyingSchema.optional[TestItOut]("name", _.name),
```

Comment:
Wasn't sure whether these should be using the `underlyingSchema` or the `schema`. I couldn't quite tell when it is appropriate to use one over the other.

Reply:
Right, so, I'd like for us to use newtypes by default, and therefore using `schema`. This implies that the fields in `TestItOut` would point to the newtypes instead of the wrapped types. 

In a subsequent PR, we'll add a `@unwrapped` trait in a subsequent PR, that will target members, so that the corresponding fields and referenced types in Scala will be the underlying type (as opposed to the newtype).

Reply:
Addressed in [aad9b81](https://github.com/disneystreaming/smithy4s/pull/333/commits/aad9b813d4ab58e98e89aa63095574d1baa1e313)